rollup-http/rollup-http-server/src/rollup/mod.rs

```diff
@@ -152,7 +152,7 @@ impl cmt_abi_address_t {
 pub struct GIORequest {
     #[validate(range(min = 0x10))] // avoid overlapping with our HTIF_YIELD_MANUAL_REASON_*
```

Comment:
I commented on the OpenAPI that we reserve 1, 2, 4. Here you are requiring > 2, but 4 is also reserved.